### DISCUSSION TITLE
i'm using Remix 0.56.0-dev... Please, how do i update

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#2761] [Q&A]
i hope updating my remix will clear all the glitches

**Comment by EngrPips** [comment]
Let's know how that plays out.

**Reply by Tarreef** [reply]
unfortunately, updating isn't the solution.
i have updated the desktop version but it's still giving me the error message.

and i still don't know how to update the web version

**Reply by EngrPips** [reply]
What is the error you are getting?

**Reply by Tarreef** [reply]
1. _**"You are using an outdated version of the zkSync plugin: v0.6.1-7, please update to v0.6.2-24 by force-refreshing the page and clearing your browser cache."**_
2. when i deploy a contract, on injected provider - METAMASK... i can only deploy it once. After that, Metamask dose'nt pop up
3. the deploy and verify button not working either
4. after deploying the revert error to the _**msg.value**_ also not poping up at the top of the screen after funding
![Screenshot_20241028_174601](https://github.com/user-attachments/assets/b6621be4-9a6a-4eb3-b122-f14cdcb08628)

![Screenshot_20241028_174019](https://github.com/user-attachments/assets/c5c83c34-fdf0-4049-b2a6-34dca2d3fc26)

**Reply by EngrPips** [reply]
I got the `error` too but I cleared the terminal and just continued with my development and everything worked fine. I must mention that working with `zksync` is optional so you can proceed and leave it if it is causing you much headache. You can also share the codebase you are working on with me so i try on my end and see how things play out

**Reply by Tarreef** [reply]
okay thank you...
but what about 2 and 4 please???

**Reply by EngrPips** [reply]
What is 2 and 4?

**Reply by EngrPips** [reply]
Honestly, working with `zksync` isn't very smooth right now. If you are having too much of a headache working with it, you can just skip the `zksync` part of the course as it is optional.

**Reply by Tarreef** [reply]
1. when i deploy a contract, on injected provider - METAMASK... i can only deploy it once. After that, Metamask dose'nt pop up
2. after deploying the revert error to the msg.value also not poping up at the top of the screen after funding

These last issues aren't related to zksync!

**Reply by EngrPips** [reply]
Use the `wallet-connect` option instead if you are having a difficult time with the `injected provider` option.
